components/profile/ProfileEmptyResult.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <section class="section has-text-centered">
+    <div class="title is-4">
+      {{ $t('profile.searchNoResultsTitle') }}
+    </div>
+    <div v-if="totalChainsLength" class="subtitle is-6">
+      {{ $t('profile.searchNoResultsText', [totalChainsLength]) }}
+    </div>
+    <NeoButton
+      v-if="totalChainsLength"
+      variant="pill"
+      :label="$t('profile.searchNoResultsButton', [chainNames[nextPrefix]])"
+      @click="switchToPrefix(nextPrefix)" />
+  </section>
+</template>
+
+<script lang="ts" setup>
+import { NeoButton } from '@kodadot1/brick'
+import { type Prefix, chainNames } from '@kodadot1/static'
+
+const props = defineProps({
+  hasAssetPrefixList: { type: Array<Prefix>, default: () => [] },
```

Comment:
prefix `has` means boolean to me, maybe change it to something more meaningfull?

Reply:
Ok. I changed it to another naming